share/config_global.toml

```
@@ -71,6 +71,18 @@ i18n = "global_settings_setting"
         optional = true
         default = ""
 
+    [security.portal]
+        [security.portal.portal_allow_edit_email]
+        type = "boolean"
```

```suggestion
        type = "boolean"
        default = 0
```
If the user change the mail left part, the autoconfig won't work anymore.

---

or we should a dynamic language to generate the autoconfig...